src/appMain/start_proprietary.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+pip list | grep -F tornado > /dev/null
+if [ $? -eq 1 ]
+  then
+    echo "Installing tornado python package"
+    sudo pip install tornado
```

Comment:
@ValeriiMalkov do not install anything during start. Specify dependency in README. 
Exit if there is no dependencies